docs/markdowns/quickstart.md

```diff
@@ -20,6 +18,38 @@ Then, install pyxet with:
 $ pip install pyxet
 ```
 
+(accountsetup)=
```

Comment:
Not sure, but this `(accountsetup)=` seems to render for me and that does not seem to be your intent

Reply:
oh? It doesn't render for me. Its supposed to create a explicit crossreference target though I am not able to figure out how to actually use it...